subprojects/core/src/test/groovy/org/gradle/internal/fingerprint/classpath/impl/ClasspathFingerprintCompareStrategyTest.groovy

```diff
@@ -32,141 +32,141 @@ class ClasspathFingerprintCompareStrategyTest extends Specification {
-        changes(strategy, includeAdded,
+        changes(strategy, shouldIncludeAdded,
             [:],
             ["one-old": fingerprint("one")]
         ) as List == [removed("one-old": "one")]
 
         where:
-        strategy   | includeAdded
+        strategy   | shouldIncludeAdded
         CLASSPATH  | true
         CLASSPATH  | false
     }
 
     @Unroll
-    def "non-trivial removal (#strategy, include added: #includeAdded)"() {
+    def "non-trivial removal (#strategy, include added: #shouldIncludeAdded)"() {
         expect:
-        changes(strategy, includeAdded,
+        changes(strategy, shouldIncludeAdded,
             ["one-new": fingerprint("one")],
             ["one-old": fingerprint("one"), "two-old": fingerprint("two")]
         ) == [removed("two-old": "two")]
 
         where:
-        strategy   | includeAdded
+        strategy   | shouldIncludeAdded
         CLASSPATH  | true
         CLASSPATH  | false
     }
 
     @Unroll
-    def "non-trivial modification (#strategy, include added: #includeAdded)"() {
+    def "non-trivial modification (#strategy, include added: #shouldIncludeAdded)"() {
         expect:
-        changes(strategy, includeAdded,
+        changes(strategy, shouldIncludeAdded,
             ["one-new": fingerprint("one"), "two-new": fingerprint("two", 0x9876cafe)],
             ["one-old": fingerprint("one"), "two-old": fingerprint("two", 0xface1234)]
         ) == [modified("two-new": "two", FileType.RegularFile, FileType.RegularFile)]
 
         where:
-        strategy   | includeAdded
+        strategy   | shouldIncludeAdded
         CLASSPATH  | true
         CLASSPATH  | false
     }
 
     @Unroll
-    def "trivial replacement (#strategy, include added: #includeAdded)"() {
+    def "trivial replacement (#strategy, include added: #shouldIncludeAdded)"() {
         expect:
-        changes(strategy, includeAdded,
+        changes(strategy, shouldIncludeAdded,
             ["two-new": fingerprint("two")],
             ["one-old": fingerprint("one")]
         ) as List == results
 
         where:
-        strategy   | includeAdded | results
+        strategy   | shouldIncludeAdded | results
         CLASSPATH  | true         | [removed("one-old": "one"), added("two-new": "two")]
         CLASSPATH  | false        | [removed("one-old": "one")]
     }
 
     @Unroll
-    def "non-trivial replacement (#strategy, include added: #includeAdded)"() {
+    def "non-trivial replacement (#strategy, include added: #shouldIncludeAdded)"() {
         expect:
-        changes(strategy, includeAdded,
+        changes(strategy, shouldIncludeAdded,
             ["one-new": fingerprint("one"), "two-new": fingerprint("two"), "four-new": fingerprint("four")],
             ["one-old": fingerprint("one"), "three-old": fingerprint("three"), "four-old": fingerprint("four")]
         ) == results
 
         where:
-        strategy   | includeAdded | results
+        strategy   | shouldIncludeAdded | results
```

Comment:
If you really need to rename the variable, make sure the data table is formatted correctly.